megamek/src/megamek/common/AmmoType.java

```diff
@@ -8585,7 +8585,7 @@ private static AmmoType createISLRT20Ammo() {
         ammo.bv = 23;
         ammo.cost = 30000;
         ammo.flags = ammo.flags.or(F_HOTLOAD);
-        ammo.setModes(new String[] { "", "HotLoad" });
+        ammo.setModes("", "HotLoad");
```

Comment:
Does setModes have an appropriate overload?

Reply:
I guess it does, but the other question is, is "setModes" used in any other location in MM/MHQ?

Reply:
It's a varargs method, so the string array creation isn't necessary.